extensions/cloud/portability-cloud-google/src/main/java/org/datatransferproject/cloud/google/GoogleJobStore.java

```diff
@@ -63,13 +67,57 @@ public GoogleJobStore(Datastore datastore, ObjectMapper objectMapper) {
     this.objectMapper = objectMapper;
   }
 
+  @VisibleForTesting
+  static String getDataKeyName(UUID jobId, String key) {
+    return String.format("%s-%s", jobId, key);
+  }
+
+  private static Map<String, Object> getProperties(Entity entity) {
```

Comment:
can this be moved back down since it is private?

Reply:
Done.